### PR TITLE
Remove a matched user

### PR DESCRIPTION
**Pull request by cgntnr**
Added ability to remove a user from My Matches. The manipulation in UserRepository is pretty similar to @cmsigrist 's code to Delete an Account, we can reuse it there. The user has a remove button besides the match's name. When clicked, a warning message is prompted and if user proceeds, that match is removed from My Matches.

Most of the manipulation is already in UserRepository however for testing the warning message, @Xelowak can address this in his testing task #241 .

### CODE REVIEW

File: app/src/main/java/ch/epfl/sdp/blindly/database/UserRepository.kt [hunk 1]
```diff
@@ -50,6 +50,30 @@ class UserRepository @Inject constructor(
...
     }
 
+    /**
+     * Removes another liked or matched user from current user.
+     *
+     * @param field field to remove a User (either from LIKES or MATCHES)
+     * @param userUid current user's UID
+     * @param matchUid matched user's UID
+     */
+    suspend fun removeMatchFromAUser(field: String, userUid: String, matchUid:String) {
```
Comment (cmsigrist) [inline app/src/main/java/ch/epfl/sdp/blindly/database/UserRepository.kt:60]: I went through your whole pull request and have not found anything to comment on... x)
So I don't know, maybe for coherence you should rename the argument userUid to userId and matchId I guess....

**Review by cmsigrist — APPROVED** [review]
Great work!

File: app/src/main/java/ch/epfl/sdp/blindly/database/UserRepository.kt (hunk 1 above)
Comment (cgntnr) [reply app/src/main/java/ch/epfl/sdp/blindly/database/UserRepository.kt:60]: Sure I can do that :D

**Review by Xelowak — APPROVED** [review]
LGTM !

**Pull request merged**